app/src/main/java/com/kickstarter/models/Amount.kt

```diff
@@ -33,7 +33,7 @@ data class Money(
 
     override fun equals(obj: Any?): Boolean {
```

Comment:
As a `data class` without any additional logic, we may not need to override `equals()` here at all.